--- a/unittesting_limo/limo_zIRLS_validation_4_Arno.m
+++ b/unittesting_limo/limo_zIRLS_validation_4_Arno.m
@@ -89,3 +89,3 @@
 % 1st level Analysis
-STUDY = pop_limo(STUDY, ALLEEG, 'method','IRLS','measure','daterp','timelim',[-50 650],'erase','on','splitreg','off','interaction','off');
+[STUDY,~,LIMOfiles] = pop_limo(STUDY, ALLEEG, 'method','IRLS','measure','daterp','timelim',[-50 650],'erase','on','splitreg','off','interaction','off');
 eeglab redraw
@@ -97,6 +97,6 @@
     fprintf('running bootstrap on subject %g\n',s)
-    LIMO = load(fullfile(STUDY.datasetinfo(s).filepath,['FaceRepetition_GLM_Channels_Time_IRLS' filesep 'LIMO.mat']));
+    LIMO = load(LIMOfiles.mat{s});
     LIMO = LIMO.LIMO; LIMO.design.bootstrap = 2500; LIMO.design.status = 'to do';
     save(fullfile(LIMO.dir,'LIMO.mat'),'LIMO'); try rmdir([LIMO.dir filesep 'H0'],'s'); end
-    H0iw{s} = fullfile(STUDY.datasetinfo(s).filepath,['FaceRepetition_GLM_Channels_Time_IRLS' filesep 'H0']);
+    H0iw{s} = fullfile(fileparts(LIMOfiles.mat{s}), 'H0');
     limo_eeg(4,LIMO)
@@ -105,3 +105,4 @@
 % use 1500 for the null and 1000 for testing
+chanlocs = fullfile(STUDY.filepath, 'derivatives', 'limo_gp_level_chanlocs.mat');
 [results.errIRLS,results.ci_errIRLS,results.maxIRLS,results.maxci_errIRLS,results.clusterIRLS,results.maxcic_errIRLS] = limo_test_glmboot(chanlocs,H0iw,'step_size',300,'Nboot',1000,'MinSamp',300);
 save results results
